tests/files-upload-access-cache.unit.test.ts

```
@@ -0,0 +1,163 @@
+
+  beforeEach(() => {
+    vi.clearAllMocks()
+    getPgClientMock.mockReturnValue({
+      end: vi.fn(() => Promise.resolve()),
+      query: vi.fn(async () => ({ rows: [] })),
+    })
+    getUserIdFromApikeyMock.mockResolvedValue('00000000-0000-0000-0000-000000000001')
+    getAppByAppIdPgMock.mockResolvedValue({ app_id: 'test-app', owner_org: 'test-org' })
+    getAppByIdPgMock.mockResolvedValue({ plan_valid: true })
+    checkPermissionPgMock.mockResolvedValue(true)
+  })
```

ESLint

[error] `beforeEach` hooks should be before any `afterEach` hooks

(test/prefer-hooks-in-order)